x/profiles/types/account_test.go

```diff
@@ -0,0 +1,310 @@
+package types_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/desmos-labs/desmos/x/profiles/types"
+)
+
+func assertNoProfileError(profile *types.Profile, err error) *types.Profile {
+	if err != nil {
+		panic(err)
+	}
+	return profile
+}
+
+func TestProfile_Update(t *testing.T) {
+	addr1, err := sdk.AccAddressFromBech32("cosmos1y54exmx84cqtasvjnskf9f63djuuj68p7hqf47")
+	require.NoError(t, err)
+
+	tests := []struct {
+		name       string
+		original   *types.Profile
+		update     *types.ProfileUpdate
+		expError   bool
+		expProfile *types.Profile
+	}{
+		{
+			name: "DoNotModify and empty fields do not update original values",
```

Comment:
why empty fields don't update the original values here? A test below say a different thing. Can we remove it?
```suggestion
			name: "DoNotModify do not update original values",
```

Reply:
Thanks, I've cleared up the test name and also the test making sure that empty values effectively update the original ones